packages/bundler/src/es6-rewriter.ts

```diff
@@ -139,28 +138,38 @@ export class Es6Rewriter {
               const newCode = serialize(newAst).code;
               return newCode;
             }
+            return null;
           }
         },
       ],
     });
-    const {code: rolledUpCode} = await rollupBundle.generate({
+    const {output} = await rollupBundle.generate({
       format: 'es',
       freeze: false,
     });
-    // We have to force the extension of the URL to analyze here because
-    // inline es6 module document url is going to end in `.html` and the file
-    // would be incorrectly analyzed as an HTML document.
-    const rolledUpUrl = getFileExtension(url) === '.js' ?
-        url :
-        appendUrlPath(url, '_inline_es6_module.js');
-    const rolledUpDocument = await this.bundler.analyzeContents(
-        rolledUpUrl as ResolvedUrl, rolledUpCode);
-    const babelFile = assertIsJsDocument(rolledUpDocument).parsedDocument.ast;
-    this._rewriteExportStatements(url, babelFile);
-    this._rewriteImportStatements(url, babelFile);
-    this._deduplicateImportStatements(babelFile);
-    const {code: rewrittenCode} = serialize(babelFile);
-    return {code: rewrittenCode, map: undefined};
+
+    if (output.length !== 1) {
+      throw new Error(`Failed to bundle.  Rollup generated ${
+          output.length} chunks or assets.  Expected 1.`);
+    }
+    for (const chunkOrAsset of output) {
+      const rolledUpCode = chunkOrAsset.code || '';
+      // We have to force the extension of the URL to analyze here because
+      // inline es6 module document url is going to end in `.html` and the file
+      // would be incorrectly analyzed as an HTML document.
+      const rolledUpUrl = getFileExtension(url) === '.js' ?
+          url :
+          appendUrlPath(url, '_inline_es6_module.js');
+      const rolledUpDocument = await this.bundler.analyzeContents(
+          rolledUpUrl as ResolvedUrl, rolledUpCode);
+      const babelFile = assertIsJsDocument(rolledUpDocument).parsedDocument.ast;
+      this._rewriteExportStatements(url, babelFile);
+      this._rewriteImportStatements(url, babelFile);
+      this._deduplicateImportStatements(babelFile);
+      const {code: rewrittenCode} = serialize(babelFile);
+      return {code: rewrittenCode, map: undefined};
+    }
+    throw new Error(`Failed to bundle.  Rollup generated no chunks or assets.`);
```

Comment:
shouldn't this be redundant with the `output.length` check above?

Reply:
Yeah but the for loop caused typescript to declare a possible undefined return.  This problem has gone away by removing the for loop and thus this throw is unneeded and now removed.